boards/arm/raytac_mdbt50q_db_33_nrf52833/doc/index.rst

```diff
@@ -0,0 +1,207 @@
+- Chip Antenna
+- Interfaces: SPI, UART, I2C, I2S, PWM, ADC, NFC, and USB
+- Highly flexible multiprotocol SoC ideally suited for Bluetooth® Low Energy, ANT+, Zigbee, Thread (802.15.4) ultra low-power wireless applications.
+- 3 User LEDs
+- 4 User buttons
+- 1 Mini USB connector for power supply and USB communication
+- SWD connector for FW programing
+- J-Link interface for FW programing
+- UART interface for UART communication
+
+Supported Features
+==================
+
+The raytac_mdbt50q_db_33_nrf52833 board configuration supports the following
+hardware features:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| ADC       | on-chip    | adc                  |
++-----------+------------+----------------------+
+| CLOCK     | on-chip    | clock_control        |
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| I2C(M)    | on-chip    | i2c                  |
++-----------+------------+----------------------+
+| MPU       | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| PWM       | on-chip    | pwm                  |
++-----------+------------+----------------------+
+| RADIO     | on-chip    | Bluetooth,           |
+|           |            | ieee802154           |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| RTT       | Segger     | console              |
++-----------+------------+----------------------+
+| SPI(M/S)  | on-chip    | spi                  |
++-----------+------------+----------------------+
+| UART      | on-chip    | serial               |
++-----------+------------+----------------------+
+| USB       | on-chip    | usb                  |
++-----------+------------+----------------------+
+| WDT       | on-chip    | watchdog             |
++-----------+------------+----------------------+
+
+Other hardware features have not been enabled yet for this board.
+See `MDBT50Q-DB-33 website`_ and `MDBT50Q-DB-33 Specification`_
+for a complete list of Raytac MDBT50Q-DB-33 board hardware features.
+
+Connections and IOs
+===================
+
+LED
+---
+
+* LED1 (green) = P0.13
+* LED2 (red) = P0.14
+* LED3 (blue) = P0.15
+
+Push buttons
+------------
+
+* BUTTON1 = SW1 = P0.11
+* BUTTON2 = SW2 = P0.12
+* BUTTON3 = SW3 = P0.24
+* BUTTON4 = SW4 = P0.25
+
+UART
+----
+* RXD = P0.08
+* TXD = P0.06
+* RTS = P0.05
+* CTS = P0.07
+
+Programming and Debugging
+*************************
+
+Applications for the ``raytac_mdbt50q_db_33_nrf52833`` board configuration can be
```

Comment:
Would also mention here that an external debugger is needed since there isn't one from what I see in the picture, and people need to know what to use and how to connect it up